### ISSUE TITLE
vagrant up removes hosts if vm is up

### ISSUE THREAD

**Issue by cgsmith** [cogitatio/vagrant-hostsupdater#88]
When you run `vagrant up` after the VM is already running it will first remove the hosts and then mention that the VM is already running.

**Comment by ryanaltvater** [comment]
I have a very similar issue. When I run `vagrant up` after a `vagrant suspend`, it removes the hosts as it's first step. If I run `vagrant reload` (or `vagrant destroy` then `vagrant up`), the issue doesn't occur and I can access the local dev URL just fine.

<img width="527" alt="screen shot 2015-12-01 at 2 00 00 pm" src="https://cloud.githubusercontent.com/assets/1336787/11510904/f09e90aa-9833-11e5-8675-473de435be08.png">

**Comment by colin-marshall** [comment]
Having the same issue here. Hosts aren't added on `vagrant up` but they are added on `vagrant reload`.

**Comment by JhovaniC** [comment]
I'm having the same issue. Hosts are removed as first step in `vagrant up` and are not added unless I do  `vagrant reload`.

**Comment by ahebrank** [comment]
I can fix this for my machine (with Vagrant 1.8, both booting fresh and with an `up` after suspend) by changing the up hook from:

```
     action_hook(:hostsupdater, :machine_action_up) do |hook|
        hook.prepend(Action::RemoveHosts)
        hook.after(Vagrant::Action::Builtin::SetHostname, Action::UpdateHosts)
      end
```

to

```
      action_hook(:hostsupdater, :machine_action_up) do |hook|
        hook.prepend(Action::RemoveHosts)
        hook.append(Action::UpdateHosts)
      end
```

I'm assuming `up` with an active hosts doesn't trigger the sethostname builtin, but I also assume there's a reason it's set up this way which is why I haven't done a PR.

**Comment by cgsmith** [comment]
Thanks @ahebrank - This fixes it for the second `up` even after suspend.  I will release the next version shortly.
